tests/models/test_tpu.py

```diff
@@ -76,7 +77,8 @@ def test_model_tpu_index(tmpdir, tpu_core):
         default_root_dir=tmpdir,
         enable_progress_bar=False,
         max_epochs=2,
-        tpu_cores=[tpu_core],
+        accelerator="tpu",
+        devices=[tpu_core],
```

Comment:
Rename the variable name from `tpu_core` to `devices` maybe for consistency?

Reply:
Got it